### ISSUE TITLE
LameXP does not encode leading silence in source WAV file

### ISSUE THREAD

**Issue by PipeOrganNorm** [lordmulder/LameXP#99]
The recent beta (4.19, RC11, build 2317) does not include the leading silence I like to include in the source WAV file.  I thought perhaps there might be a LAME switch that would affect this behavior, but could not find anything in the LAME documentation.  Neither could I find anything in the documentation included with LameXP, or any settings that could change this action.  I didn't know what else to do, the only difference being the "debug console", which I enabled on v4.18 (and disabled on 4.19).  Consistently, 4.19 would erroneously remove the silence, and 4.18 would properly include it in the encoded output.

I have used LameXP for years, and truly appreciate your work.  It's a terrific program.  My hat is off to your for all your efforts.

Thanks

**Comment by lordmulder** [comment]
Well, keep in mind that LameXP is just a GUI front-end. If you encode WAV file to MP3 fromat with LameXP, it will invoke `lame.exe` and pass your input WAV file as-is. There is ***nothing*** in LameXP that would "trim" the silence at the beginning of your file.

I don't see how the behavior should have changed between LameXP v4.18 and v4.19. If at all, it could be because the LAME binary that ships with LameXP has been updated. But then again, the development of LAME is pretty much "dead" for a long time. Someone is still maintaining the code, applying a few minor bug fixes every now and  then. But that is pretty much it. There were **no** major changes in years. In fact, the version of LAME included with LameXP v4.18 and v4.19 is the same - `3.100.1`. For LameXP 4.19, I re-built the latest LAME code from SVN, just to be sure. But there certainly was **no** change that should have caused ***this***.

Also, to the best of my knowledge, LAME does **not** contain any feature that would  would "trim" the silence at the beginning of your file. In fact, it does quite the opposite and ***adds*** some padding (silence) at the beginning and at the end:
```
1.  Why is a decoded MP3 longer than the original .wav file?
2.  Why does LAME add silence to the beginning each song?
3.  Why does LAME add silence to the end of each song?
```
:point_right: **https://lame.sourceforge.io/tech-FAQ.txt**

--------

So how *exactly* did you determine this? And you are 100% certain that you are *not* "fooled" by the MP3 ***decoder*** ???

AFAIK, some MP3 decoders try to compensate for the "encoder delay" (added silence by encoder) in order to allow for what this known as "seamless playback". What you are experiencing might actually be exactly this: The MP3 ***decoder*** removing the silence.

**Comment by lordmulder** [comment]
No further details was provided, so closing this one for now.
